[Feat 9] agregar contributions y actualizar documentación de instalación (#10)
taecontrol/larastats-docs: docs/contributions.md

---
id: contributions
slug: /contributions
sidebar_position: 13
---

# Contributions Guide

At this time we are currently accepting the current forms of contributions:

* Bug reports.
* Pull requests for bug fixes.

We are not accepting these forms of contributions:

* New Features.

## Bug Reports

> In case you report a bug, your issue should contain a title and a clear description
> of the issue. You should also include revelant information and code samples to
> replicate the bug and develop a fix as easy as posible.

Reviewer: ```suggestion
If you encounter a bug, please make sure to include a descriptive title and a clear description of the issue in your report. It is also helpful to provide any relevant information and code samples that can help the maintainers replicate the bug and develop a fix as easily as possible.
```

## Development

### Installation

> For install a development environment you can check the manual installation
> [here](https://docs.moonguard.dev/installation#manual-installation).

Reviewer: ```suggestion
To install a development environment, you can refer to the manual installation guide found [here](https://docs.moonguard.dev/installation#manual-installation).
```
